### DISCUSSION TITLE
Checking Version

### DISCUSSION THREAD

**Discussion by nebhead** [tsightler/ring-mqtt#484] [Support Q&A]
This may be a dumb question, but is there an easy way to determine the version of ring-mqtt docker container / image from the command line?  I currently use portainer, and when I click into the container, I see the following:

![image](https://user-images.githubusercontent.com/19876886/196062432-ad20e5e7-479e-4183-82b1-4a0437075b69.png)

Looks like there is a v5.0.3 there, but I'm not sure if that is truly relevant.  

I have pulled the image at the command line and it says I have the latest.  Just wanted to double check.  

Thanks in advance!

**Comment by tsightler** [comment]
I actually have no idea if there's a way to do this from Docker itself, but you can query the Docker registry API with the following:

`curl -L -s 'https://registry.hub.docker.com/v2/repositories/tsightler/ring-mqtt/tags?page_size=1024'|jq '."results"[]["name"]'`

This will output all available tags (i.e. versions), if you only want to see the latest version you can reduce the page_size to 2.  Someone else may have a better way to get this information, but this is what I know works.

**Reply by nebhead** [reply]
Thanks!  Well, I am nearly certain I have the latest (5.0.5).  Was just trying to confirm which image was actually on my server.  Anyway, all good.

**Reply by tsightler** [reply]
The image version you show above indicates 5.0.3, but perhaps you have upgraded since then or perhaps Portainer UI didn't refresh if you did a manual pull.  I haven't used Portainer in quite a while, but I would expect it to show the correct image information.  Also, the code does show in the banner during startup.

**Reply by nebhead** [reply]
Yeah, I actually used the stacks feature to pull the latest in Portainer, but it probably hasn't refreshed the information for some reason. I'll have to check what the banner says when it boots up.  Thanks again!
